detox/android/detox/src/full/java/com/wix/detox/adapters/server/DetoxServerInfo.kt

```diff
@@ -0,0 +1,19 @@
+package com.wix.detox.adapters.server
+
+import com.wix.detox.LaunchArgs
+import com.wix.detox.common.DetoxErrors
+
+class DetoxServerInfo internal constructor(launchArgs: LaunchArgs = LaunchArgs()) {
+    val serverUrl: String? = launchArgs.detoxServerUrl
+    val sessionId: String? = launchArgs.detoxSessionId
+
+    init {
+        if (serverUrl == null || sessionId == null) {
+            throw DetoxErrors.DetoxRuntimeException("Detox-server connection info is missing - breaking bad! url=$serverUrl, sessionId=$sessionId")
```

Comment:
While we are already rewritting things here, let's sync with ios implementation. On ios, there is a default value - localhost: 8099 and sessionId=package-name.

Reply:
Is that so! Sure, an easy fix. Is there logic to handle a case when only either of the two is null?

Reply:
Done